src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -76,7 +76,7 @@ export class SignifyApi {
     const signifyName = utils.uuid();
     const operation = await this.signifyClient
       .identifiers()
-      .create(signifyName, this.getCreateAidOptions());
+      .create(signifyName); //, this.getCreateAidOptions());
```

Comment:
Closer to IIW we need to decide if we'll use cardano-backer, witnesses or nothing (lower demo risk)

Reply:
okey, we can just remove cardano-backer for the demo